x-pack/plugins/secops/public/apps/start_app.tsx

```diff
@@ -7,6 +7,7 @@
 import { createHashHistory } from 'history';
 import React from 'react';
 import { ApolloProvider } from 'react-apollo';
+import { Provider as ReduxStoreProvider } from 'react-redux';
```

Comment:
I take it this is for naming convention to keep it similar in naming to `ApolloProvider` above? Normally this is taboo, but it makes sense here. Just a note, no changes requested.

Reply:
I'm following the conventions of the `infra` project pretty closely. The screenshot below shows the `infra` version on the left, and the `secops` version on the right:

![image](https://user-images.githubusercontent.com/4459398/48216525-2ac99200-e342-11e8-8a02-c401d05ad234.png)